ndbench-cli/build.gradle

```diff
@@ -19,6 +19,9 @@ sourceCompatibility = 1.8
 
 repositories {
     mavenCentral()
+    flatDir {
```

Comment:
Is there a reason why we do not need `mavenCentral()` anymore?

Reply:
fixed